Missing closing bracket

gurevichmark opened the pull request:
Build error caused by #5116 
Missing closing bracket

cxhong commented:
Good!

whowutwut commented:
Was this caught by Travis and how do we prevent this from happening again or could we? Did sandbox build catch it?

gurevichmark commented:
This happened when I moved one of the functions from `xcatd.pm` to `Utils.pm`. While removing the function from the end of `xcatd.pm`, I deleted one line too many. Thinking that no functional changes were done, I did not retest.

whowutwut commented:
Does a sandbox build catch it? On commit push the Travis should do this... did that fail?

gurevichmark commented:
The build was successful, this mistake prevents `xcatd` from starting.
